Settings.xm

```diff
@@ -49,16 +49,6 @@ extern BOOL hideShorts(); // this is also not necessary since you have IsEnabled
     selectBlock:^BOOL (YTSettingsCell *cell, NSUInteger arg1) {
         return [%c(YTUIUtils) openURL:[NSURL URLWithString:@"https://github.com/arichorn/CercubePlusExtra/releases/latest"]];
     }];
-	
-	YTSettingsSectionItem *hideShorts = [[%c(YTSettingsSectionItem) alloc]initWithTitle:LOC(@"HIDE_SHORTS_VIDEOS") titleDescription:LOC(@"HIDE_SHORTS_VIDEOS_DESC")];
-    hideShorts.hasSwitch = YES;
-    hideShorts.switchVisible = YES;
-    hideShorts.on = [[NSUserDefaults standardUserDefaults] boolForKey:@"hideShorts_enabled"];
-    hideShorts.switchBlock = ^BOOL (YTSettingsCell *cell, BOOL enabled) {
-        [[NSUserDefaults standardUserDefaults] setBool:enabled forKey:@"hideShorts_enabled"];
-        return YES;
-    };
-    [sectionItems addObject:main];

```

Comment:
[REPOSTED REPLY]
Well.. the reason why HideShorts was the only option to not be in the new CercubePlus Settings was because of this following issue https://github.com/arichorn/CercubePlusExtra/issues/9
It’s fine though I guess I can work with it. 👍